conjur/controller/init_controller.py

```diff
@@ -192,4 +194,4 @@ def ensure_overwrite_file(cls, config_file: str):
         force_overwrite = input(f"File {config_file} exists. "
                                 f"Overwrite? yes/no (Default: yes): ").strip()
         if force_overwrite != '' and force_overwrite.lower() not in VALID_CONFIRMATIONS:
-            raise Exception(f"Not overwriting {config_file}")
+            raise ConfirmationException(f"Not overwriting {config_file}")
```

Comment:
same as the error above we shouldn't be using exceptions to handle our valid code execution

Reply:
disagree with you here. See my comment above https://github.com/cyberark/conjur-api-python3/pull/306#discussion_r650665223